#!/usr/bin/env python
PACKAGE = "lidar_tracking"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("is_displaying_pcl", bool_t,    0, "output pcl or not",    True)

gen.add("window_width", int_t,    0, "window_width to ground filter",    3, 3,   100)
gen.add("cov_thres", double_t,    0, "cov_thres to ground filter",    0.1, 0.0,   1.0)
gen.add("grid_length", double_t,    0, "grid to ground filter",    0.1, 0.0,   1.0)
gen.add("grid_width", double_t,    0, "grid to ground filter",    0.1, 0.0,   1.0)
gen.add("area_length", double_t,    0, "select pcl area",    3.6, 1.0,   20.0)
gen.add("area_width", double_t,    0, "select pcl area",    5.2, 3.0,   20.0)

exit(gen.generate(PACKAGE, "tracking_node", "ParamLoc"))
